config_example.py

```diff
@@ -1,4 +1,5 @@
 locations_url = "https://host/v1/locations"
 token_api = "https://api.oregonstate.edu/oauth2/token"
 client_id = ""
-client_secret = ""
\ No newline at end of file
+client_secret = ""
+campuses = ['corvallis', 'cascades', 'hmsc', 'other']
```

Comment:
Just curious, is there any reason you define campuses list in config file instead of in get_locations_csv.py?